src/Data/Fin/Properties.agda

```diff
@@ -849,6 +872,27 @@ pigeonhole (s≤s (s≤s m≤n)) f with any? (λ k → f zero ≟ f (suc k))
   suc i , suc j , i≢j ∘ suc-injective ,
   punchOut-injective (f₀≢fₖ ∘ (i ,_)) _ fᵢ≡fⱼ
 
+-- Cantor-Schröder-Bernstein for finite sets
+
+injective⇒≤ : {k l : ℕ} {f : Fin k → Fin l} → Injective _≡_ _≡_ f → k ℕ.≤ l
+injective⇒≤ {ℕ.zero} {l} {f} H = z≤n
```

Comment:
Can we call this `inj` instead of `H` everywhere? It's a little bit more meaningful when you're reading the proofs.